docs/admin/machine.rst

```diff
@@ -395,6 +395,8 @@ ModernMT
                 | ``key`` | API key |  |
                 +---------+---------+--+
```

Comment:
The new `context_vector` field should be documented here. You can generate the table for documentation from the code using `./manage.py list_machinery`.